File: sdk/typescript/src/integration.ts
export async function execute(connectorType: string, operation: string, payload: any, baseUrl?: string) {
  const url = `${baseUrl || process.env.NEXT_PUBLIC_INTEGRATION_URL || ''}/connectors/execute`;
  const res = await fetch(url, {
    method: 'POST',
    headers: { 'Content-Type': 'application/json' },
    body: JSON.stringify({ connector_type: connectorType, operation, payload })
  });
  const data = await res.json();
  if (!res.ok) throw new Error(data.detail || 'Failed to execute');
  return data.result;
}

File: sdk/python/apex_sdk/integration.py
"""Python SDK for the integration service."""

from __future__ import annotations

from typing import Any, Dict
from .client import APEXClient


class IntegrationAPI:
    def __init__(self, client: APEXClient) -> None:
        self._client = client

    async def execute(self, connector_type: str, operation: str, payload: Dict[str, Any]) -> Any:
        req = {
            "connector_type": connector_type,
            "operation": operation,
            "payload": payload,
        }
        res = await self._client.post(self._client.integration_url, "/connectors/execute", req)
        res.raise_for_status()
        return res.json().get("result")